pkg/processor/build/builder.go

```diff
@@ -76,6 +76,9 @@ type Builder struct {
 		// a list of commands that execute when the processor is built
 		commandsToRunDuringBuild []string
 
+		// list of files to copy
+		filesToCopy []string
```

Comment:
`filePathsToCopy`
I know it's pedantic but I had a hell of a time working with the previous iteration on the builder due to never knowing if a variable held the absolute directory, the absolute full path, the relative path, the file name, the file contents or the file handle

Reply:
Sorry about that, will fix